client/my-sites/sharing/buttons/preview-action.jsx

```diff
@@ -1,40 +1,41 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { omit } from 'lodash';
 
-module.exports = React.createClass( {
+export default React.createClass( {
 	displayName: 'SharingButtonsPreviewAction',
 
 	propTypes: {
-		active: React.PropTypes.bool,
-		position: React.PropTypes.oneOf( [
+		active: PropTypes.bool,
+		position: PropTypes.oneOf( [
 			'top-left',
 			'top-right',
 			'bottom-left',
 			'bottom-right'
 		] ),
-		icon: React.PropTypes.string,
-		onClick: React.PropTypes.func
+		icon: PropTypes.string,
+		onClick: PropTypes.func
 	},
 
-	getDefaultProps: function() {
+	getDefaultProps() {
 		return {
 			active: true,
 			position: 'top-left',
-			onClick: function() {}
+			onClick: () => {}
```

Comment:
Hi! Would it be better practice to use lodash/noop here? I've seen it used elsewhere for defaults :)